File: src/main/java/pl/training/workshop/m6/s13_extractcomposite/step1/Film.java
package pl.training.workshop.m6.s13_extractcomposite.step1;

/** Krok 1: bez zmian - liść - pojedynczy film. */
public record Film(String title, int minutes) implements ProgramItem {
    @Override
    public String describe() {
        return title + " (" + minutes + " min)";
    }
}

File: csharp/src/Training.Workshop/M6/S13ExtractComposite/Step1/Film.cs
namespace Training.Workshop.M6.S13ExtractComposite.Step1;

/// <summary>Krok 1: bez zmian - liść - pojedynczy film.</summary>
public sealed record Film(string Title, int Minutes) : IProgramItem
{
    public string Describe()
    {
        return Title + " (" + Minutes + " min)";
    }
}
